### ISSUE TITLE
[Accessbility]A11y_Java Tooling for IntelliJ/Eclipse - New Features_Remote Debugging-Debugger_Keyboard: Unable to switch between Debugger & Console Tabs using keyboard

### ISSUE THREAD

**Issue by InduPriya1805** [microsoft/azure-tools-for-java#3599]
**GitHub Tags:** #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #A11yMAS; #A11yTCS; #C+E-H; #DesktopApp; #Win10; #JavatoolingforIntelliJ/Eclipse; #Keyboard; #MAS2.1.1; 

[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
Download and Install IntelliJ IDEA from https://www.jetbrains.com/idea/download/#section=windows
IntelliJ IDEA 2019.2.1
Build #IU-192.6262.58

**OS Details:**
OS: Windows (Version 1903, Build: 18362.113)

**MAS Violated:** 2.1.1

**Note:
Similarly user is unable to switch between [spark on HDInsight] Spark_WasbIOTest & Submit sample.Spark Core_WasbIOTest**

**Repro Steps:**
1)Launch IntelliJ IDEA
2)Tab till create new project-->Azure Spark/HDInsight-->Spark project with Samples(Scala) 
3)Tab till next button and press enter
4)Fill all the details correctly and Tab till finish button and press enter
5)Tab till <Project Name>-->src-->main-->scala-->sample-->SparkCore_wasbloTest and open it
6)Tab till Edit Configurations menu(select the icon in the upper-right corner)
7)Navigate to Advanced Configuration (Remote Debugging) section and Press the shortcut key to check Enable Spark remote debug
8)Fill Serure Shell (SSH) User Name and SSH password
9)Sent some breakpoints in SparkCore_wasbloTest code and tab till sparkjobdebug and press enter
10)Press cntrl+tab to navigate to Debug Remote Spark Job in Cluster section and verify user is able to switch between Debugger & Console Tabs using keyboard or not

**Actual Result:**
Unable to switch between Debugger & Console Tabs using keyboard

**Expected Result:**
User should be able to  switch between Debugger & Console Tabs using keyboard

**User Impact:**
Keyboard users will find difficult in working if they are unable to switch between the tabs  using keyboard

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Attachment:**
[Bug_3599_unbale to switch between debug and console using keyboard.pptx](https://github.com/microsoft/azure-tools-for-java/files/3641871/Bug_3599_unbale.to.switch.between.debug.and.console.using.keyboard.pptx)

**Comment by t-rufang** [comment]
Note:
The first bug is we can't switch between `Debugger` and `Console` tab. Since debug window is supported by IntelliJ, we won't fix this one. 
![image](https://user-images.githubusercontent.com/32627233/65598876-fe0c8080-dfce-11e9-886a-46672a2ba259.png)

The second bug is we can't switch between the `Debug Remote Spark Job in Cluster`. We need to figure out if this is our code bug or if this feature is not supported by IntelliJ.
![image](https://user-images.githubusercontent.com/32627233/65598978-47f56680-dfcf-11e9-8a6f-184d875f49d2.png)

**Comment by t-rufang** [comment]
The job view page is wrote as static web pages with the help of JavaFX WebView which has no support for Accessibility. Please refer to JDK issue [JDK-8177382](https://bugs.openjdk.java.net/browse/JDK-8177382)

**Comment by Alexandria-R** [comment]
This has been approved to close by SME.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749702
